seahorn/jobs/array_list_back/CMakeLists.txt

```diff
@@ -6,3 +6,12 @@ target_compile_definitions(array_list_back
   MAX_INITIAL_ITEM_ALLOCATION=9223372036854775808ULL)
 sea_attach_bc_link(array_list_back)
 sea_add_unsat_test(array_list_back)
+
+# klee
+add_executable(array_list_back.klee ${AWS_C_COMMON_ROOT}/source/array_list.c
+  aws_bounded_array_list_back_harness.c)
+target_compile_definitions(array_list_back.klee
+  PUBLIC MAX_ITEM_SIZE=2
```

Comment:
avoid adding definitions per file.

Reply:
I made a new marco for klee. It is now fixed.